arviz/tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+"""Configuration for test suite."""
+# pylint: disable=redefined-outer-name
+import os
+import pytest
+
+
+def pytest_addoption(parser):
+    """Definition for command line option to save figures from tests."""
+    parser.addoption("--save", nargs="?", const="test_images",
+                     help="Save images rendered by plot")
+
+
+@pytest.fixture(scope="session")
+def save_figs(request):
+    """Enable command line switch for saving generation figures upon testing."""
+    fig_dir = request.config.getoption("--save")
+
+    if fig_dir is not None:
+        # Try creating directory if it doesn't exist
+        print("Saving generated images in {}".format(fig_dir))
+        try:
+            os.mkdir(fig_dir)
+            print("Directory {} created".format(fig_dir))
+
+        except FileExistsError:
+            print("Directory {} exists".format(fig_dir))
+
```

Comment:
Use 

`os.makedirs(path, exist_ok=True)`

No need to make a try-except here.

This will also enable multiple new subdirs.